casper/src/test/scala/coop/rchain/casper/api/BlocksResponseAPITest.scala

```diff
@@ -104,11 +104,39 @@ class BlocksResponseAPITest
   }
   implicit val turanOracleEffect: SafetyOracle[Id] = SafetyOracle.turanOracle[Id]
 
-  "getBlocksResponse" should "return only blocks in the main chain" in {
+  "showMainChain" should "return only blocks in the main chain" in {
     val blocksResponse =
-      BlockAPI.getBlocksResponse[Id](syncId, casperRef, logEff, turanOracleEffect, blockStore)
-    val blocks = blocksResponse.blocks
+      BlockAPI.showMainChain[Id](Int.MaxValue)(
+        syncId,
+        casperRef,
+        logEff,
+        turanOracleEffect,
+        blockStore
+      )
     blocksResponse.length should be(5)
-    blocks.length should be(5)
+  }
+
+  "showBlocks" should "return all blocks" in {
+    val blocksResponse =
+      BlockAPI.showBlocks[Id](Int.MaxValue)(
+        syncId,
+        casperRef,
+        logEff,
+        turanOracleEffect,
+        blockStore
+      )
+    blocksResponse.length should be(8) // TODO: Switch to 4 when we implement block height correctly
```

Comment:
What is this TODO message about?

Reply:
This is what we discussed in discord chat a few days ago. We just increment blockNumber by 1 instead of max(parents) + 1 in tests.